core/src/sql/subquery.rs

```diff
@@ -32,6 +33,7 @@ pub enum Subquery {
 	Insert(InsertStatement),
 	Define(DefineStatement),
 	Remove(RemoveStatement),
+	Rebuild(RebuildStatement),
```

Comment:
@rushmorem @DelSkayn I have incremented the revision, I think this is not necessary when we add an enum at the end?

Reply:
It can change the enum size, so probably is necessary